apps/storybook/src/SelectionTool.stories.tsx

```diff
@@ -122,24 +113,33 @@ export const Persisted: Story<
 
       <SelectionTool
         modifierKey={selectionModifierKey}
-        onSelectionStart={() => setPersistedSelection(undefined)}
-        onSelectionEnd={setPersistedSelection}
+        onSelectionStart={() => setPersistedDataSelection(undefined)}
+        onSelectionEnd={(selection) =>
+          setPersistedDataSelection(selection.data)
+        }
       >
-        {({ data: [dataStart, dataEnd] }) => (
-          <SelectionRect startPoint={dataStart} endPoint={dataEnd} />
+        {(selection) => (
+          <SvgRect coords={selection.world} fill="teal" fillOpacity="0.3" />
         )}
       </SelectionTool>
 
-      {persistedSelection && (
-        <SelectionRect
-          startPoint={persistedSelection.data[0]}
-          endPoint={persistedSelection.data[1]}
-        />
+      {persistedDataSelection && (
+        <DataToWorld coords={persistedDataSelection}>
+          {(...worldCoords) => (
+            <SvgRect coords={worldCoords} fill="teal" fillOpacity="0.6" />
+          )}
+        </DataToWorld>
```

Comment:
This shows how to use `DataToWorld` along with the new `SvgRect` component. Without it, since the story component renders `VisCanvas`, I wouldn't be able to retrieve `dataToWorld` from the context without creating a new component.